test/public_plans_interceptor_test/public_plans_test.go

```diff
@@ -116,20 +118,43 @@ var _ = Describe("Service Manager Public Plans Interceptor", func() {
 				IsCatalogPlanPublicFunc: func(broker *types.ServiceBroker, catalogService *types.ServiceOffering, catalogPlan *types.ServicePlan) (b bool, e error) {
 					return *catalogPlan.Free, nil
 				},
-				SupportedPlatforms: func(ctx context.Context, plan *types.ServicePlan, repository storage.Repository) ([]string, error) {
+				SupportedPlatforms: func(ctx context.Context, plan *types.ServicePlan, repository storage.Repository) (map[string]*types.Platform, error) {
 					return service_plans.ResolveSupportedPlatformIDsForPlans(ctx, []*types.ServicePlan{plan}, repository)
 				},
+				TenantKey: "tenant",
 			}).OnTxBefore(interceptors.BrokerCreateCatalogInterceptorName).Register()
-
+			_, err := smb.EnableMultitenancy("tenant", func(request *web.Request) (string, error) {
+				extractTenantFromToken := multitenancy.ExtractTenantFromTokenWrapperFunc("zid")
```

Comment:
looks like this is the 4th occurrence of this callback function in the tests, maybe it is time to extract and re-use?